clusterloader2/testing/load/config.yaml

```diff
@@ -490,6 +490,10 @@ steps:
     objectBundle:
     - basename: big-deployment
       objectTemplatePath: deployment.yaml
+      templateFillMap:
+        ReplicasMin: {{MultiplyInt $BIG_GROUP_SIZE 0.5}}
```

Comment:
This is not needed for the delete step. Note that this test runs continuously without any issues - https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-cos-master-scalability-100

Reply:
it really does need it , if do the test in local cluster, or it will fail with "cannot convert nil to int64"

Reply:
We run this test periodically (tens of times a day on different cluster sizes). It is NOT needed.